openfisca_core/formulas.py

```diff
@@ -676,9 +676,10 @@ def neutralize_column(column):
         )
 
 
-def new_filled_column(base_function = UnboundLocalError, calculate_output = UnboundLocalError,
+def new_filled_column(__doc__ = None, __module__ = None,
+        base_function = UnboundLocalError, calculate_output = UnboundLocalError,
         cerfa_field = UnboundLocalError, column = UnboundLocalError, comments = UnboundLocalError,
-        __doc__ = None, __module__ = None,
+        default = UnboundLocalError,
```

Comment:
If I recall well: we had a problem when neutralizing column that had a default value which was not the parent Column default value. @eraviart can provide you more details ;-)